src/metabase/driver/generic_sql/query_processor.clj

```diff
@@ -341,9 +341,10 @@
   "Set the timezone for the current connection."
   [driver settings connection]
   (let [timezone (:report-timezone settings)
-        sql      (sql/set-timezone-sql driver)]
-    (log/debug (u/pprint-to-str 'green [sql timezone]))
-    (jdbc/db-do-prepared connection [sql timezone])))
+        sql      (sql/set-timezone-sql driver)
+        replaced (clojure.string/replace sql "?" (str "'" (re-matches #"[A-z\/_]+" timezone) "'"))]
```

Comment:
https://github.com/metabase/metabase/wiki/Metabase-Clojure-Style-Guide#require-namespaces-you-use